.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
TIL! The js-helpers repo did not have dependabot working, so I assumed that this one did not either. It could be that this was using dependabot preview, in which case dependabot would have broken.